app/Http/Resources/UserResource.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class UserResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return array
+     */
+    public function toArray($request) {
+        return [
+            'id'                => (int) $this->id,
+            'displayname'       => (string) $this->name,
```

Comment:
```suggestion
            'displayName'       => (string) $this->name,
```

I'm not sure, but is this maybe better?